lms/envs/common.py

```diff
@@ -3021,3 +3021,20 @@
 ############## Settings for LMS Context Sensitive Help ##############
 
 DOC_LINK_BASE_URL = None
+
+########################## Course Discovery #######################
+LANGUAGE_MAP = {'terms': {lang: display for lang, display in ALL_LANGUAGES}, 'name': 'Language'}
```

Comment:
I'm not familiar with this feature, so it's hard to say. The fact that it was only present in devstack settings suggests that this is only meant to be configured in devstacks. If that's not true, then maybe it should be moved.